torch/_inductor/mkldnn.py

```diff
@@ -131,8 +132,10 @@ def _update_module_params(self, conv, unary, input_size):
                 self.stride,
                 self.dilation,
                 self.groups,
-                tuple(guard_int(x) for x in input_size),
-            ),
+                input_size,
+            )
+            if input_size is not None
+            else self.weight.to_mkldnn(),
```

Comment:
Regarding the `input_size`, should we still keep `guard` for the static shape?

Reply:
yes, kept.